test/functional/test-custom-element.js

```diff
@@ -1012,88 +1011,110 @@ describes.realWin('CustomElement', {amp: true}, env => {
       }).to.throw(/Must never be called in template/); });
     });
 
+    // TODO(alabiaga): Fixes #19752. The window.matchMedia call in the code
+    // path of element.applySizesAndMediaQuery is not behaving consistently
+    // in headless mode, thus we mock the calls here. This is fine as we are
+    // not testing window behavior.
```

Comment:
Why a TODO? Also we should move the explanation to where we stub `matchMedia`.

Reply:
I wanted to follow up on why this why happening but a comment should suffice.